about.html

```diff
@@ -0,0 +1,127 @@
+  <link href="https://fonts.googleapis.com/css2?family=Rubik+Dirt&display=swap" rel="stylesheet">
+  <link
+    href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&display=swap"
+    rel="stylesheet">
+  <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+</head>
+
+<body>
+  <header>
+    <div>
+      <ul>
+        <li><i class="fa-brands fa-facebook-f"></i></li>
+        <li><i class="fa-brands fa-twitter"></i></li>
+        <li>English</li>
+        <li>My Page</li>
+        <li>Logout</li>
+      </ul>
+    </div>
+    <div>
+      <img src="./assets/logo-talent-land.png" alt="">
+      <span id="openOnClick"><a href="javascript:void(0);" class="icon" onclick="myFunction()">&#9776</a></span>
+      <nav>
+        <ul>
+          <li><a href="./index.html">Home</a></li>
+          <li>Program</li>
+          <li>Join</li>
+          <li>Sponsor</li>
+          <li>News</li>
+        </ul>
+      </nav>
+      <div class="dropdown">
+        <ul id="options" class="dropdown-content">
+          <li><a href="./index.html">Home</a></li>
+          <li>Program</li>
+          <li>Join</li>
+          <li>Sponsor</li>
+          <li>News</li>
+        </ul>
+      </div>
+      <div>
+        <p>CC Campaing</p>
+      </div>
+    </div>
+  </header>
+  <main>
+    <section class="about_headline">
+      <div class="container">
+        <div>
+          <h1>Talent Land Mexico 2023</h1>
+        </div>
+        <div>
+          <p>Talent Land 2023 will be dedicated to the development of talent, so that the participants can meet with
+            experts from different areas who will share their experience and knowledge.
+            This event is a place for creative minds, entrepreneurs and young talents to find new ideas, new people and
+            new technologies to improve their personal and professional lives.
+            Talent Land 2023 is underway and it will be unique.</p>
+        </div>
+        <div>
+          <p>Please contact us per email for any further questions about CC Talent Land Mexico.</p>
+          <p>info@talent-network.org</p>
+        </div>
+      </div>
+    </section>
+
+    <section class="about_talent container">
+      <h2>Jalisco Talent Land </h2>
+      <hr>
+      <p>Since the first Talent Land we managed to bring together creativity and technology, for the next 2023, the
+        machinery has been set in motion to create new emotions, promote the development of talent and promote
+        entrepreneurship among young people.
+        With the same entrepreneurial spirit that has allowed us to carry out unforgettable experiences on a global
+        level, in April 2023 we will discover new ways to change the world with talent.
+        “Talent Land 2023: the future is yours”
+        Get ready!</p>
+      <div>
+        <img src="./assets/logo-talent-land.png" alt="">
+      </div>
+    </section>
```

Comment:
- [ ] This section does not look like the original design. There's a missing content somewhere here and there's small underline below subtitle on this section which is missing on your design, Kindly follow the design content by content, line by line as to not omit any content. 
### Your Design
![your-small-underline](https://user-images.githubusercontent.com/26597432/233479477-119e5c1b-beb9-4dad-9d6d-9cc163cb749a.png)

### Figma Design
![temp-about-small-underline](https://user-images.githubusercontent.com/26597432/233479539-dbe7a5c5-7868-4137-87c9-5ca6b34e31f4.png)